src/navigation/iconsConfigs.js

```diff
@@ -0,0 +1,26 @@
+export const iconConfigs = {
+    'Home':{
+        iconName: 'home',
+    },
+    'All books':{
+        iconName: 'book',
+    },
+    'List Of Books':{
+        iconName: 'bookmark',
+    },
+    'Book Shop':{
+        iconName: 'shopping-basket',
+    },
+    size: 20,
```

Comment:
`key_name test` instead `Book Shop And Santa` - is pretty, and my best knowledge (cosmetic)

Reply:
key_name === page title. its easier for me to understand

Reply:
Then call it `tabBarIconsConfig`. Otherwise, it should be attached to navigation context

Reply:
fixed